FakeItEasy.ruleset

```diff
@@ -5,4 +5,17 @@
     <Rule Id="CA1020" Action="None" />
     <Rule Id="CA1305" Action="None" />
   </Rules>
+  <Rules AnalyzerId="StyleCop.Analyzer" RuleNamespace="StyleCop.Analyzer">

```

Comment:
I see this section and StyleCop.Analyzer**s** in the IDE. Any idea why both? The latter has many more entries.